Views/UserControls/ChefLookingAtRecipeView.xaml

```diff
@@ -35,6 +39,14 @@ Frost. Remove from the oven and allow to cool for about 10 minutes, remove from
                 HorizontalAlignment="Right"
                 Margin="0 0 15 10"
                 Padding="35 8"
-                FontSize="14"/>
+                FontSize="14"
+                Command="{Binding CompleteDishesCommand}">
+            <Button.CommandParameter>
```

Comment:
If we bind whole ViewModel we avoid MyMultiValueConverter
`
CommandParameter ="{Binding}"
`